pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -972,4 +974,11 @@ public static class Range {
   public static class IdealState {
     public static final String HYBRID_TABLE_TIME_BOUNDARY = "HYBRID_TABLE_TIME_BOUNDARY";
   }
+
+  public static class RewriterConstants {
```

Comment:
I wonder why should this be defined in `CommonConstants` where we typically define Broker or Server instance level config constants ? Can this be taken inside rewriter as public constants ?

Reply:
putting this in rewriter will make it not accessible from SPI

Reply:
I see. Hmm we should try to fix this in follow-ups